charts/sourcegraph/templates/syntactic-code-intel/worker.Deployment.yaml

```diff
@@ -0,0 +1,126 @@
+{{- if .Values.syntacticCodeIntel.enabled -}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    description: Handles syntactic code intelligence indexing
+  labels:
+    {{- include "sourcegraph.labels" . | nindent 4 }}
+    {{- if .Values.syntacticCodeIntel.labels }}
+      {{- toYaml .Values.syntacticCodeIntel.labels | nindent 4 }}
+    {{- end }}
+    deploy: sourcegraph
+    app.kubernetes.io/component: syntactic-code-intel
+  name: {{ .Values.syntacticCodeIntel.name }}
+spec:
+  minReadySeconds: 10
+  replicas: {{ .Values.syntacticCodeIntel.replicaCount }}
```

Comment:
is the new code intel worker horizontally scalable? reason asking is `worker` appears to be but it's not.

will we see weird behavior if two instances are active at once? conflicted indexing results?

Reply:
> is the new code intel worker horizontally scalable

It should be, yes. Similar to the precise codeintel worker, the identity of the worker instance is used to "lock" certain records during processing. As it's part of the design of dbworker (which we ultimately delegate to), I believe this shouldn't be a problem.